### PR TITLE
Setup/testing

### PR DESCRIPTION
**Pull request by DagmarTimmreck**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,13 +6,21 @@
   "scripts": {
     "start": "node index.js",
     "dev": "nodemon index.js",
```
Comment (Oxyrus) [inline package.json:8]: Same here

File: package.json [hunk 2]
```diff
@@ -6,13 +6,21 @@
   "scripts": {
     "start": "node index.js",
```
Comment (Oxyrus) [inline package.json:7]: Change to use babel

**Review by Oxyrus — COMMENTED** [review]
The application won't work unless the scripts used to start it run babel.

File: package.json (hunk 2 above)
Comment (DagmarTimmreck) [reply package.json:7]: opted for module.exports

**Review by Oxyrus — APPROVED** [review]

**Pull request merged**